cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -1574,12 +1437,10 @@ pTxBuildOutputOptions =
   pCalculatePlutusScriptCost :: Parser TxBuildOutputOptions
   pCalculatePlutusScriptCost =
     OutputScriptCostOnly
-      <$> Opt.strOption
-        ( Opt.long "calculate-plutus-script-cost"
-            <> Opt.metavar "FILE"
-            <> Opt.help "(File () Out) filepath of the script cost information."
-            <> Opt.completer (Opt.bashCompleter "file")
-        )
+      <$> File
+      <$> parseFilePath
+        "calculate-plutus-script-cost"
+        "(File () Out) filepath of the script cost information."
```

Comment:
I know that it was copied from before, but I'm not sure about this haskell bit in the documentation: `(File () Out)`. Seems odd, and not really informative for people not familiar with cardano-cli codebase.

Reply:
Good point, I changed the message :+1: